selfdrive/car/gm/values.py

```diff
@@ -10,9 +10,13 @@ class CAR:
   MALIBU = "CHEVROLET MALIBU PREMIER 2017"
   ACADIA = "GMC ACADIA DENALI 2018"
   BUICK_REGAL = "BUICK REGAL ESSENCE 2018"
+  BOLT = "CHEVROLET BOLT"
+  EQUINOX = "CHEVROLET EQUINOX"
 
 SUPERCRUISE_CARS = [CAR.CADILLAC_CT6]
 
+NO_ASCM_CARS = [CAR.BOLT, CAR.EQUINOX]
```

Comment:
I don't think categorizing NO_ASCM_CARS is the best representation by car is the best representation.

LKA_ONLY or NO_ACC is probably a general categorization. Since variable is just used to prevent OP from generating gas/brake commands for cars that can't use them.

This makes sense as written for the Bolt as there is no ACC version, however the Volt comes in both an ACC+LKA version as well as a LKA only version. So a Volt could fit in either category depending on how its optioned.

As I stated in my comment in carcontroller.py I think the answer here is to have a 'LongitudinalControl' parameter, now that Toyota/Honda has firmware based fingerprinting, I think we should look towards implementing something similar in GM and with firmware version we could probably tell the difference between a LKA only Volt and an ACC+LKA Volt.